### DISCUSSION TITLE
Using uv

### DISCUSSION THREAD

**Discussion by fepegar** [microsoft/cookie-doh#51] [General]
Hi and thanks for this great repo.

I have written a version using uv (instead of PDM, pipx and pyenv):
- https://github.com/fepegar/cookie-doh/pull/1

The CI passes and it seems to be working locally.

If you think this might be interesting, I'm happy to open a PR here.

**Comment by jzazo** [comment]
If you want, create a PR and I will merge into cookie-doh. I would like to make some suggestions though. Thank you!
